Signal/src/call/NonCallKitCallUIAdaptee.swift

```diff
@@ -51,13 +55,35 @@ class NonCallKitCallUIAdaptee: CallUIAdaptee {
         notificationsAdapter.presentMissedCall(call, callerName: callerName)
     }
 
+    func answerCall(localId: UUID) {
+        CallService.signalingQueue.async {
+            guard let call = self.callService.call else {
+                assertionFailure("\(self.TAG) in \(#function) No current call.")
+                return
+            }
+
```

Comment:
Is the parameter the id of the call?  Should we assert that it matches the id of the "current" call?